src/stores/batchActions.ts

```diff
@@ -4,6 +4,8 @@ import ErrorReporter from 'lib/errors/ErrorReporter';
 import { FlashMessage, messageTTL, messageType } from 'lib/flashMessage';
 import { IOAuth2Provider } from 'lib/OAuth2/OAuth2';
 import RoutePath from 'lib/routePath';
+import { HttpClientImpl } from 'model/clients/HttpClient';
+import { selfSubjectRulesReview } from 'model/services/mapi/authorization';
```

Comment:
@axbarsan could you help me figure out how to mock this in a test? Since it's part of the main batchActions, I need a way to avoid having this try to make an actual API call, since we don't have a nock for the MAPI requests and want to avoid doing that right

Reply:
For now I think we can mock the function in all tests, like we do with the info and metadata requests.

- https://github.com/giantswarm/happa/blob/master/testUtils/modelMocks.js
- https://github.com/giantswarm/happa/blob/master/testUtils/mockHttpCalls/metadata.ts
- https://github.com/giantswarm/happa/blob/779eac7d32679c7a64fbaf54a9cb6ab435a6d910/__tests__/AdminLogin.tsx#L24

Also, you might want to only do this request for MAPI users, and not for oldschool GS users